packages/SwingSet/src/kernel/state/storageWrapper.js

```diff
@@ -175,6 +175,8 @@ export function buildCrankBuffer(storage) {
 }
 
 export function cacheHeuristic(key) {
+  return false; // XXX disabling for now, since the cache appears both useless and expensive
```

Comment:
cool, good, but let's add an Issue to delete this entirely eventually

Reply:
Gone in the latest rev